Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# syntax=docker/dockerfile:1
+
+######################### BUILD STAGE #############################
+# This stage builds the HarbourBridge binary using its source code.
+
+# Add golang as the base image for the build stage.
+FROM golang:1.18.3 AS build
+
+# Set the default work directory as “/hbBin” in the container.
+WORKDIR /hbBin
+
+# Copy HarbourBridge’s source code into the working directory.
+COPY . ./
+# Download HarbourBridge’s dependencies mentioned in the go.mod file.
+RUN go mod tidy 
+# Compile the source code and store the binary in the work directory /hbBin.
+RUN make build-static  
+
+######################### Release Stage ###########################
+# This stage consists of the GCloud SDK and the HarbourBridge binary.
+
+# Add the Ubuntu base image from the Google Container Registry.
+FROM gcr.io/cloud-marketplace/google/ubuntu1804@sha256:d9fe979ab72ff02f5b5d95c59cda6681aa73039fbf9b46596935267cf26c692e
+# Set the default working directory.
+# This directory binds with the host directory that is to store the output files generated by HarbourBridge.
+WORKDIR /harbourBridge/harbour_bridge_output
+# Create directory to store the HarbourBridge binary. 
+RUN mkdir /harbourBridge/bin
+# Create directory to store the dump files. This directory binds with the host directory that contains the dump files of the source database.
+RUN mkdir /harbourBridge/sourceDump
+# Copy the HarbourBridge binary from the build stage's hbBin directory to the release stage's harbourBridge/bin directory.
+COPY --from=build /hbBin/harbourbridge /harbourBridge/bin
+
+# Add Python3 to support GCloud SDK.
+RUN apt-get update
+RUN apt-get install -y python3.10
```

Comment:
Why is this needed? What versions of `python` does `gCloud SDK` support?

Reply:
GCloud SDK needs Python2.7.9 or higher. Refer - [Python Requirement](https://cloud.google.com/sdk/docs/install#:~:text=The%20Google%20Cloud%20CLI%20requires%20Python%203%20(3.5%20to%203.8%2C%203.7%20recommended)%20and%20Python%202%20(2.7.9%20or%20higher).).